### PR TITLE
[Components] Axonaut #6418

### PR DESCRIPTION
**Pull request by lcaresia**
### <samp>🤖 Generated by Copilot at 1badbec</samp>

This pull request removes the `axonaut` component, which was no longer used or maintained, and updates the `pnpm-lock.yaml` file to use the latest versions of the dependencies. This reduces the codebase size and improves the reliability and functionality of the project.

### <samp>🤖 Generated by Copilot at 1badbec</samp>

> _To fix bugs and boost performance_
> _We updated some dependencies_
> _In the `pnpm-lock.yaml` file_
> _We changed the versions with style_
> _And enhanced some nice features hence_

### <samp>🤖 Generated by Copilot at 1badbec</samp>

*  Delete `components/axonaut/.gitignore` and `components/axonaut/app/axonaut.app.ts` files ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-1a307715cdc56bfe74b7e23f4e36a384ed2374f1f0d462af9f7a6c2abbb09dd7), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-0721373deac590847bada23f7875762295e4f9b0c21a3f957af8ed6ae9261054))
*  Update `fast-glob` dependency to version 3.3.1 in `pnpm-lock.yaml` file to fix bugs and improve performance ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL16809-R16794), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL22402-R22387), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL24121-R24106))
*  Update `@types/node` dependency to version 20.4.4 in `pnpm-lock.yaml` file to add and update type definitions for Node.js ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL17028-R17013), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL17994-R17979), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18133-R18118), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18289-R18274), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18304-R18289), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18421-R18406), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18462-R18447), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18579-R18564), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18710-R18695), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18771-R18756), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18874-R18859), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18924-R18909), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL18945-R18930), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL21879-R21864), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL22141-R22126), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL22161-R22146), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL24065-R24050))
*  Update `postcss` and `postcss-safe-parser` dependencies to version 8.4.27 in `pnpm-lock.yaml` file to fix bugs and improve performance ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL21915-R21900), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL21921-R21906), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL21953-R21939), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL24138-R24125))
*  Update `whatwg-fetch` dependency to version 3.6.17 in `pnpm-lock.yaml` file to fix a bug related to the `Content-Type` header ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL17994-R17979), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL25669-R25655))
*  Update `@sindresorhus/is` dependency to version 5.6.0 in `pnpm-lock.yaml` file to add support for `BigInt` and `SharedArrayBuffer` types ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL17028-R17013))
*  Update `word-wrap` dependency to version 1.2.5 in `pnpm-lock.yaml` file to fix a bug related to the `break` option ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL21371-R21356), [link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL25805-R25791))
*  Update `@putout/plugin-conditions` dependency to version 2.2.0 in `pnpm-lock.yaml` file to add support for `LogicalExpression` and `ConditionalExpression` nodes ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL22304-R22289))
*  Update `aws-sdk` dependency to version 2.1420.0 in `pnpm-lock.yaml` file to add and update features and services of the AWS SDK ([link](https://github.com/PipedreamHQ/pipedream/pull/7356/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL23651-R23636))

### CODE REVIEW

File: components/axonaut/axonaut.app.mjs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+export default {
+  type: "app",
+  app: "axonaut",
+  propDefinitions: {
+    companyId: {
+      label: "Company ID",
+      description: "The company ID",
+      type: "string",
+      async options() {
+        const companies = await this.getCompanies();
```
Comment (vellames) [inline components/axonaut/axonaut.app.mjs:12]: I am wondering if the API provides pagination for get companies. Looks like we are fetching everything or just the first page.

File: components/axonaut/axonaut.app.mjs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+        return companies.map((company) => ({
+          value: company.id,
+          label: company.name,
+        }));
+      },
+    },
+    employeeId: {
```
Comment (vellames) [inline components/axonaut/axonaut.app.mjs:20]: I am wondering if the API provides pagination for get companies. Looks like we are fetching everything or just the first page.

File: components/axonaut/package.json [hunk 3]
```diff
@@ -2,15 +2,17 @@
   "name": "@pipedream/axonaut",
   "version": "0.0.2",
```
Comment (vellames) [inline components/axonaut/package.json:3]: `0.1.0` since we are creating new components?

**Review by vellames — CHANGES_REQUESTED** [review]
Hi there!
I left just a few comments

File: components/axonaut/axonaut.app.mjs (hunk 1 above)
Comment (lcaresia) [reply components/axonaut/axonaut.app.mjs:12]: I didn't find pagination in the docs.

File: components/axonaut/axonaut.app.mjs (hunk 2 above)
Comment (lcaresia) [reply components/axonaut/axonaut.app.mjs:20]: I didn't find pagination in the docs.

**Review by vellames — APPROVED** [review]

**Review by vellames — DISMISSED** [review]

**Review by vellames — APPROVED** [review]

**Pull request merged**